units/user/default.target

```diff
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Default
+Description=Main user target
```

Comment:
we'd usually capitalize this as "Main User Target" like in a headline. The other unit files usually used such upper case capitalization